From d6df77b2d68c1433af0fef15e5889c6f08c4d521 Mon Sep 17 00:00:00 2001
From: Janne Grunau <j@jannau.net>
Date: Mon, 1 Apr 2024 08:09:27 +0200
Subject: [PATCH] dracut: Install grep, mkdir as functions.sh deps

Missing grep in the initramfs was observed with dracut on gentoo.

Signed-off-by: Janne Grunau <j@jannau.net>
---
 dracut/modules.d/99asahi-firmware/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut/modules.d/99asahi-firmware/module-setup.sh b/dracut/modules.d/99asahi-firmware/module-setup.sh
index 9018db5..db6ef99 100755
--- a/dracut/modules.d/99asahi-firmware/module-setup.sh
+++ b/dracut/modules.d/99asahi-firmware/module-setup.sh
@@ -30,7 +30,7 @@ install() {
     asahiscriptsdir="/usr/share/asahi-scripts"
     inst_dir $asahiscriptsdir
     $DRACUT_CP -R -L -t "${initdir}/${asahiscriptsdir}" "${dracutsysrootdir}${asahiscriptsdir}"/*
-    inst_multiple cpio cut dirname modprobe mount seq sleep umount
+    inst_multiple cpio cut dirname grep mkdir modprobe mount seq sleep umount
     inst_hook pre-udev 10 "${moddir}/load-asahi-firmware.sh"
     inst_hook cleanup 99 "${moddir}/install-asahi-firmware.sh"
 }
